disnake/i18n.py

```diff
@@ -192,6 +193,9 @@ def _upgrade(
         return self
 
 
+Localised = Localized
```

Comment:
this should be documented in the docstring for `Localized`

Reply:
abcb75c5e6af34d92cf9ab431f48908a875f3e5f